sample-apps/python-manual-instrumentation-sample-app/README.md

```diff
@@ -0,0 +1,21 @@
+## Python Opentelemetry Auto-instrumentation Sample App
+
+### Getting Started:
+
+#### Local
+
+```
+# create venv
+python3 -m venv .
+
+source bin/activate
+
+# install requirements
+pip install --no-cache-dir -r requirements.txt
+
+# run app with environment variables set.
+python app.py
+```
+
+#### Docker
+Build the image using the dockerfile and run the image in a container.
```

Comment:
can you write the command to build? I know it is simple, but still when looking for a sample app, people just want to copy and paste the commands.